fixed compile time errors and warnings for IAR-EWARM projects

aaronjense opened the pull request:
**Changes**:
The sp.c file was removed and the following were added: sp_arm32.c, sp_arm64.c, sp_32.c, sp_64.c and sp_int.c

evp.c and bio.c were removed from compiling since a warning was given that they don't need to be compiled seperately from ssl.c

**Status**:
The following is the only warning given now and is due to a test environment.

wolfssl\wolfcrypt\src\random.c(2027) : Warning[Pa183]: #warning directive: "write a real random seed!!!!, just for testing now"

dgarske commented on IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewp:
```diff
@@ -1021,7 +1021,7 @@
                 <option>
                     <name>IarchiveOutput</name>
-                    <state>C:\Users\synecdoche\Documents\GitHub\wolfssl\IDE\IAR-EWARM\Projects\lib\ewarm\Exe\wolfSSL-Lib.a</state>
+                    <state>C:\Users\Administrator\Desktop\aaron-testing-IAR\wolfssl\IDE\IAR-EWARM\Projects\lib\ewarm\Exe\wolfSSL-Lib.a</state>
```
Can you update this to use the `$PROJ_DIR$` or some other variable?

dgarske commented on IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewp:
```diff
@@ -1,2104 +1,2277 @@
+                </option>
+                <option>
+                    <name>CCPosIndRwpi</name>
+             
```
Can we update this to use `$PROJ_DIR$` or some other variable?

aaronjense commented:
I made the requested changes to wolfSSL-Lib.ewp by changing the hard-coded path to "$PROJ_DIR$\..\lib\ewarm\Exe\wolfSSL-Lib.a".   I probably could have done "$PROJ_DIR$\ewarm\Exe\wolfSSL-Lib.a", but I went with the more verbose path.

dgarske commented:
@aaronjense : The project file also updates the required IAR version. Is it possible you can make the changes to the ewp files manually to avoid upgrading the project files? I.E just modify the XML to add/remove the files used. Unless there is a good reason to keep the .ewd files I think we should stop tracking them in Git.

aaronjense commented:
I made the minimal changes. 

There is a warning message when opening the workspace file for the first time.  Everything is fine once there is a rebuild.

Changed project file will force a full rebuild: C:\Users\Administrator\Desktop\aaron-testing\IAR-EWARM\wolfssl\IDE\ 
IAR-EWARM\Projects\test\wolfCrypt-test.ewp 
Changed project file will force a full rebuild: C:\Users\Administrator\Desktop\aaron-testing\IAR-EWARM\wolfssl\IDE\ 
IAR-EWARM\Projects\lib\wolfSSL-Lib.ewp
